lua/telescope/_extensions/lazy/config.lua

```diff
@@ -26,6 +26,9 @@ M.defaults = {
     width = 0.5,
     height = 0.5,
   },
+  open_in_browser_opts = {
```

Comment:
I think this should be made more future proof and generic by having opts for each specific action under the same key.

```lua
  actions_opts = {
    open_in_browser = {
      auto_close = true,
    },
  },

```

Reply:
Should `auto_close` be true or false?

Setting to `true` would change the default behavior, but that might make sense.

Reply:
I believe it's best to keep it as `false` to avoid altering any existing behavior and causing inconvenience.